lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -173,9 +173,10 @@ class ReportUIFeatures {
       });
 
     tablesWithUrls.forEach((tableEl, index) => {
+      const numRows = this._getUrlItems(tableEl).length;
       const thirdPartyRows = this._getThirdPartyRows(tableEl, this.json.finalUrl);
-      // No 3rd parties, no checkbox!
-      if (!thirdPartyRows.size) return;
+      // If all or none of the rows are 3rd party, no checkbox!
+      if (thirdPartyRows.size === numRows || !thirdPartyRows.size) return;
```

Comment:
I think we should be able to nuke `thirdPartyFilterAuditExclusions` with just this :)

Reply:
I've found a few cases where this doesn't work, where resources are being loaded from a different subdomain on the same domain.

Reply:
Ah yeah if they're intermixed it would be weird to show it good call 👍